base/complex.jl

```diff
@@ -476,6 +476,8 @@ function inv(w::ComplexF64)
         return ComplexF64(1/c, -0.0)
     elseif d === -0.0
         return ComplexF64(1/c,  0.0)
+    if d == 0.0
+        return ComplexF64(1/c, -d)
```

Comment:
this suggestion didn't apply properly. This branch was supposed to replace the `if` and `elseif` above it.